assets/src/dashboard/app/views/myStories/storyGrid.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import {
+  StoryGrid as _StoryGrid,
+  CardGridItem,
+  CardTitle,
+  CardPreviewContainer,
+  PreviewPage,
+} from '../../../components';
+import { storiesPropType } from '../../../propTypes';
+
+const StoryGrid = ({ filteredStories }) => (
+  <_StoryGrid>
```

Comment:
I get this, but underscore components could get confusing down the road. It might be good to name this `StoryGridContainer` or the top one `StoryGridView`.